### ISSUE TITLE
2D-UNet training notebook doesn't work for Colab

### ISSUE THREAD

**Issue by qin-yu** [constantinpape/torch-em#69]
For code in https://colab.research.google.com/github/constantinpape/torch-em/blob/main/experiments/2D-UNet-Training.ipynb

```python
!nvidia-smi
```

> NVIDIA-SMI has failed because it couldn't communicate with the NVIDIA driver. Make sure that the latest NVIDIA driver is installed and running.

```python
!conda install -c conda-forge torch_em "pyyaml<5.0"
```

> Collecting package metadata (current_repodata.json): done
> Solving environment: failed with initial frozen solve. Retrying with flexible solve.
> Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
> Collecting package metadata (repodata.json): done
> Solving environment: / 
> The environment is inconsistent, please check the package plan carefully
> The following packages are causing the inconsistency:
> 
>   - conda-forge/linux-64::mamba==0.8.0=py37h7f483ca_failed with initial frozen solve. Retrying with flexible solve.
> Solving environment: / 
> Found conflicts! Looking for incompatible packages.
> This can take several minutes.  Press CTRL-C to abortfailed
> 
> UnsatisfiableError: The following specifications were found
> to be incompatible with the existing python installation in your environment:
> 
> Specifications:
> 
>   - pyyaml[version='<5.0'] -> python[version='>=3.10,<3.11.0a0|>=3.9,<3.10.0a0']
> 
> Your python: python=3.7
> 
> If python is on the left-most side of the chain, that's the version you've asked for.
> When python appears to the right, that indicates that the thing on the left is somehow
> not available for the python version you are constrained to. Note that conda will not
> change your python version to a different minor version unless you explicitly specify
> that.
> 
> The following specifications were found to be incompatible with each other:
> 
> Output in format: Requested package -> Available versions
> 
> Package setuptools conflicts for:
> python=3.7 -> pip -> setuptools
> torch_em -> pytorch -> setuptools[version='<59.6|>=41.0.0']
> 
> Package pypy3.6 conflicts for:
> torch_em -> python[version='>=3.6'] -> pypy3.6[version='7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*']
> pyyaml[version='<5.0'] -> python[version='>=3.6,<3.7.0a0'] -> pypy3.6[version='7.3.0.*|7.3.1.*|7.3.2.*|7.3.3.*|>=7.3.3|>=7.3.2|>=7.3.1']
> 
> Package _libgcc_mutex conflicts for:
> python=3.7 -> libgcc-ng[version='>=9.4.0'] -> _libgcc_mutex[version='*|0.1',build='conda_forge|main']
> pyyaml[version='<5.0'] -> libgcc-ng[version='>=7.3.0'] -> _libgcc_mutex[version='*|0.1',build='conda_forge|main']The following specifications were found to be incompatible with your system:
> 
>   - feature:/linux-64::__glibc==2.27=0
>   - torch_em -> pytorch -> __cuda
>   - torch_em -> pytorch -> __glibc[version='>=2.17|>=2.17,<3.0.a0']
> 
> Your installed version is: 2.27

**Comment by constantinpape** [comment]
> ```
> !nvidia-smi
> 
> NVIDIA-SMI has failed because it couldn't communicate with the NVIDIA driver. Make sure that the latest NVIDIA driver is installed and running.
> ```

This is probably caused because you don't have a colab session with gpu support and should be fixed if you change this.


> ```python
> !conda install -c conda-forge torch_em "pyyaml<5.0"
> ```

The failure seems to be caused by the pyyaml dependency not being available for python 3.7 anymore (this used to  work....).
You can try to remove the pyyaml dependency, i.e. just run 
> ```python
> !conda install -c conda-forge torch_em
> ```

Hopefully this fixes the error. (Pinning the pyyaml version was necessary at some point due to issues with the bioimageio libraries; I hope these are fixed now and it should just work). 

Please give it a try and let me know if this fixes the error.

**Comment by constantinpape** [comment]
I have updated the notebooks now in order to use the pre-installed pytorch version. It's working for me, so I am closing this. Feel free to reopen if you run into any other issues with colab.
